xml/deploy_additional.xml

```diff
@@ -300,6 +300,55 @@
       </listitem>
      </itemizedlist>
     </note>
+    <important>
+     <title>Incorrectly reported image sizes</title>
```

Comment:
I would have a title like "Disabling emulate_legacy_capacity option" or just "emulate_legacy_capacity option".

Reply:
Perhaps just append it, e.g. `Incorrectly reported image sizes (emulate_legacy_capacity option)`?